common/transport/mqtt/src/main/java/org/thingsboard/server/transport/mqtt/adaptors/JsonMqttAdaptor.java

```diff
@@ -207,6 +207,7 @@ public static JsonElement validateJsonPayload(UUID sessionId, ByteBuf payloadDat
         try {
             return new JsonParser().parse(payload);
         } catch (JsonSyntaxException ex) {
+            log.error("Payload is in incorrect format: {}", payload);
```

Comment:
Please change this to WARN or INFO. This may DDoS our demo server :-)

Reply:
done